### DISCUSSION TITLE
1.2.4 no display but 1.2.3-2 Normal recognition screen

### DISCUSSION THREAD

**Discussion by GengGode** [rustdesk/rustdesk#7855] [Q&A]
1.2.4 Tell me there is no monitor, but it is available when connecting 1.2.3-2

have monitor: 1.2.3-2->1.2.3-2
no display: 1.2.3-2->1.2.4

**Comment by fufesou** [comment]
Hi, @GengGode thanks for your feedback.

Can you please also upload a screenshot or a video? This way I can know what error message it reports.

I've tryied 1.2.3-2 (Win) -> 1.2.4 (Win). It connect normally.

**Reply by fufesou** [reply]
Please also try 1.2.4 -> 1.2.4, thanks.

**Reply by GengGode** [reply]
I tested again and everything was normal, 1.2.3-2->1.2.4 and 1.2.4->1.2.4
I was unable to connect normally before, so I tried using RDP remotely. This screen appeared and I closed RDP to try to connect normally. Then a window popped up displaying "no display", but now everything is normal. Previously, it was possible that the monitor was dormant due to waiting. Thank you for your help
<img width="424" alt="e16febb289b2cc6702d58d8a5e5ce82e" src="https://github.com/rustdesk/rustdesk/assets/47711102/b193539b-7918-4cfb-bc1d-f08903fd2b97">
